### ISSUE TITLE
Keep spell checker exceptions file clean

### ISSUE THREAD

**Issue by locks** [emberjs/guides#1105]
We should have a script that's run, preferably with the tests, to make sure that every word in the spell checker exception file is actually used.

While this may lead to some churn in the future if said word is reintroduced by another commit, I think the benefit of making the writer more aware outweighs it.

**Comment by jackfranklin** [comment]
@locks I'm going to take a stab at this if that's OK, hoping to get a PR in soon :+1:

**Comment by hd719** [comment]
Hello I am new to Open source can I work on this?

**Comment by acorncom** [comment]
@hd719 absolutely!  Feel free to ping us on the `#-team-learning` channel on the Ember Community Slack if you run into issues too ...

**Comment by hd719** [comment]
Hello @aaron0browne since this is my one of my first contributions how would I actually start and try to tackle this problem. I forked over the repo. and got the dev. env working smoothly. 

-Thank you.

**Comment by EndangeredMassa** [comment]
Can this be closed? It appears to have been resolved by: https://github.com/emberjs/guides/pull/1943
